src.ts/utils/web.ts

```diff
@@ -11,7 +11,29 @@ export interface ConnectionInfo {
     headers?: { [key: string]: string | number };
 }
 
-const fetch = (typeof window === 'undefined' || window.name === 'nodejs') ? require('node-fetch') : window.fetch;
```

Comment:
Feel like makes sense to extract into separate package. Like e.g. in wallet I use this package:
https://github.com/nearprotocol/send-json/blob/master/index.js

We should just extend it with other goodies.